ct.yaml

```diff
@@ -2,4 +2,5 @@ remote: origin
 chart-dirs:
   - ./
 target-branch: master
-helm-extra-args: --timeout 8m
\ No newline at end of file
+helm-extra-args: --timeout 8m
+check-version-increment: false
```

Comment:
@excieve this should address the issue we discussed regarding bumping chart versions on every PR.